mute_app.sh

```diff
@@ -62,9 +62,11 @@ fi
 
 # muting...
 if [[ "$2" == "mute" ]]; then
-    pacmd set-sink-input-mute "$current_index" 1 > /dev/null 2>&1
+   pacmd set-sink-input-mute "$current_index" 1 > /dev/null 2>&1
+   # amixer set -D pulse Master toggle -q
 else
-    pacmd set-sink-input-mute "$current_index" 0 > /dev/null 2>&1
+   pacmd set-sink-input-mute "$current_index" 0 > /dev/null 2>&1
+   # amixer set -D pulse Master toggle -q
```

Comment:
Please remove unused commented code and fix the indentation to 4 spaces. Same for line 66.